recently added webrtc code breaks build

### Describe the bug

```
rosettadrone\app\src\main\java\sq\rogue\rosettadrone\plugins\WebRTCStreaming.java:60: error: m_model is not public in DroneModel; cannot be accessed from outside package
            if (pluginManager.mainActivity.mModel.m_model == null) {
                                                 ^
rosettadrone\app\src\main\java\sq\rogue\rosettadrone\plugins\WebRTCStreaming.java:68: error: m_model is not public in DroneModel; cannot be accessed from outside package
                        pluginManager.mainActivity.mModel.m_model, stunServer);
                                                         ^
```

### Steps To Reproduce

Checkout master, build

### Screenshots

_No response_

### Drone Model

```Markdown
mavic 2 pro
```


### Smartphone Model

```Markdown
- Model:
- Android version:
```


### Additional context

tried several different gradle and java versions to see if it's a compatibility issue, all fail consistently with the same error

--- a/app/src/main/java/sq/rogue/rosettadrone/DroneModel.java
+++ b/app/src/main/java/sq/rogue/rosettadrone/DroneModel.java
@@ -204,7 +204,7 @@ public class DroneModel implements CommonCallbacks.CompletionCallback {
 
     public float mission_alt = 0;
 
-    Model m_model;
+    public Model m_model;
     Camera m_camera;
     Aircraft m_aircraft;
 

